src/Object/Contact.php

```diff
@@ -822,7 +822,33 @@ public static function getAccountType(array $contact)
 	}
 
 	/**
-	 * @brief Updates the avatar links in a contact only if needed
+	 * @brief Blocks a contact
+	 *
+	 * @param int $uid
+	 * @return bool
+	 */
+	public static function block($uid)
```

Comment:
isn't this method rather a model method?

Reply:
Yes, but at the moment we don't have a separate Contact model class. I'm holding off because @annando isn't thrilled by my `ContactModel/ContactObject` suggested discrimination. Until we decide something I will put model method in `Object\Contact` as it currently exists.